frontend/packages/console-shared/src/utils/__tests__/resource-utils.spec.ts

```diff
@@ -69,11 +51,7 @@ const knativeKeys = [Keys.REVISIONS, Keys.KNATIVECONFIGS, Keys.KSROUTES];
 
 describe('TransformResourceData', () => {
   it('should create Deployment config Items', () => {
-    const transformedData = createOverviewItemsForType(
-      'deploymentConfigs',
-      MockResources,
-      knativeOverviewResourceUtils,
-    );
+    const transformedData = createOverviewItemsForType('deploymentConfigs', MockResources);
```

Comment:
So now this is not testing the knative utils scenario anymore?

Reply:
Removed the `createOverviewItemsForType` util as it is not used anywhere.